cmd/node/factory/structs.go

```diff
@@ -2324,7 +2324,7 @@ func generateInMemoryAccountsAdapter(
 	hasher hashing.Hasher,
 	marshalizer marshal.Marshalizer,
 ) (state.AccountsAdapter, error) {
-	cacheSize := 100
+	cacheSize := uint(100)
```

Comment:
Should this be hardcoded here?

Reply:
This function is called only when creating genesis header. After that, the real accounts adapter will be used.